Containerfile

```diff
@@ -28,7 +28,7 @@ RUN $package_manager -y install dnf-plugins-core && \
     xmlto \
     && $package_manager --enablerepo=PowerTools --setopt=PowerTools.module_hotfixes=true install javapackages-local \
     && $package_manager -y clean all \
-    && pip3 install ipdb pytest flexmock
+    && pip3 install ipdb pytest flexmock prometheus_client
```

Comment:
Uh, just realised that this is going to be needed in `files/Containerfile.worker`, too, at least until #114 is fixed.

Reply:
done